lib/WorkflowRnaseq.groovy

```diff
@@ -164,6 +165,24 @@ class WorkflowRnaseq {
         }
     }
 
+    //
+    // Function that parses Salmon quant 'meta_info.json' output file to get inferred strandedness
+    //
+    public static String getSalmonInferredStrandedness(json_file) {
+        def lib_type = new JsonSlurper().parseText(json_file.text).get('library_types')[0]
+        def strandedness = 'reverse'
+        if (lib_type) {
+            if (lib_type in ['U', 'IU']) {
+                strandedness = 'unstranded'
+            } else if (lib_type in ['SF', 'ISF']) {
+                strandedness = 'forward'
+            } else if (lib_type in ['SR', 'ISR']) {
+                strandedness = 'reverse'
+            }
+        }
+        return strandedness
+    }
```

Comment:
@rob-p does this logic look ok when inferring strandedness from the `meta_info.json` file?

Reply:
Hi @drpatelh,

  Sorry for the delayed reply (end of the semester gets wild). Yes, this looks right!  Awesome to finally have this in :).

Reply:
No worries! Awesome, thanks!